Doc/library/os.path.rst

```diff
@@ -488,6 +488,41 @@ the :mod:`glob` module.)
       Accepts a :term:`path-like object`.
 
 
+.. function:: splitroot(path)
+
+   Split the pathname *path* into a triad ``(drive, root, tail)`` where:
```

Comment:
Oh, I quite like "triad" here — for me it's quite clear and succinct.

Reply:
I picked "triad" rather than "triple" because `splitdrive()` uses "pair" rather than "double". But honestly I don't mind "triple" if that's more obvious.

I don't like `3-tuple` because the bit in monospace already makes it obvious that it's a tuple. We could make that work by adjusting it as follows:

> Split the *path* into a 3-item tuple of strings *drive*, *root* and *tail*, where:

I'm borrowing "n-item tuple" from the `urllib.parse` docs as I have them to hand.

Reply:
```suggestion
   Split the pathname *path* into a triple ``(drive, root, tail)`` where:
```

Reply:
I like "3-item tuple" or "triad". Not so keen on "3-tuple" or "triple".

Reply:
When I search the web for "triad" used with "tuple", I do come across "triad" to mean a 3-item ordered sequence. So maybe it's just me. It's not as common as "triple" or "3-tuple", however. I also like "3-item tuple".

The term "tuple" is a generalization of the words double, triple, quadruple, quintuple, and so on, which are based on Latin words that combine a number (e.g. duo, tres, quattor, quintus) and "-plus" (i.e. English "-fold").

Reply:
The docs for `os.walk()` use "3-tuple" and "triple". There's plenty of other occurences of "triple"; less of "3-tuple".

What don't you like about "triple", Alex?

Reply:
I've seen "triple" used in the CPython docs in a few other places, but I've never seen it used as a noun outside the CPython docs. I know it _can_ be used as a noun, but I think technical documentation should generally strive to be clear to laypeople, not just people who are used to reading technical documentation. (There are obviously limits to how far you should take that principle, but it seems like a good principle to aspire towards.) Part of that, in my opinion, means using words in the same way that people will be used to seeing them used in contexts outside of technical documentation.

Reply:
Anyway, it sounds like we all like "3-item tuple", so why don't we just go with that? :)

Reply:
```suggestion
   Split the pathname *path* into a 3-item tuple ``(drive, root, tail)`` where:
```